src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/OrdersController.cs

```diff
@@ -129,7 +129,7 @@ public async Task<IActionResult> CreateOrderAsync(CreateOrderModel model)
             return CreatedAtAction(
                 nameof(GetAsync).TrimAsync(),
                 new { callOffId = order.CallOffId.ToString() },
-                null);
+                new { orderId = order.CallOffId.ToString() });
```

Comment:
is that right? callOffId and orderId are the same value?

Reply:
Yeah, it's due to bad use of domain language.

Outside of code it's always a call off ID but there's places in code (such as the order form) where it's still referred to as an order ID. It would be good to fix that but probably better to treat it as a tech debt task.